packages/core/helper-plugin/lib/src/icons/SortIcon/SortIcon.stories.mdx

```diff
@@ -0,0 +1,27 @@
+<!--- SortIcon.stories.mdx --->
```

Comment:
is there really any point in having a story for a svg?

Reply:
Yes it is better having one than none.